### DISCUSSION TITLE
v1.30.0_46-dev

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#760] [Announcements]
## v1.30.0_46-dev

This release includes an experimental feature on the mobile app that will help with performance improvement for viewing a large amount of assets (ten of thousands). Please visit the `Settings` page, turn on the feature, then sign out and sign back in for the new feature to take effect. Thanks @matthinc for this spectacular improvement work.

We are looking forward to your feedback on this feature.

<img src="https://user-images.githubusercontent.com/27055614/192897396-9a57176a-0562-428d-82f8-08773c9f4eb0.png" width="400" />

## What's Changed
* feat(mobile): Improve timeline performance on mobile - experimental by @matthinc in https://github.com/immich-app/immich/pull/710
* feat(server): Provide a sensible dumpDirectory for the local-reverse-geocoder module by @nebulade in https://github.com/immich-app/immich/pull/759

## New Contributors
* @nebulade made their first contribution in https://github.com/immich-app/immich/pull/759

**Full Changelog**: https://github.com/immich-app/immich/compare/v1.29.6_45-dev...v1.30.0_46-dev

<hr /><em>This discussion was created from the release <a href='https://github.com/immich-app/immich/releases/tag/v1.30.0_46-dev'>v1.30.0_46-dev</a>.</em>

**Comment by Shisharo** [comment]
Sorry if I disturb, but can i ask why you attach only some times the correspondent APK file to the releases?

**Reply by alextran1502** [reply]
Not at all, since the `deviceId` might change if you use the APK instead of installing from the AppStore, I am starting to hesitate to include the APK in the release. I includes the APK now just in case you need it.
